tests/virtualization/xen/open_vm_tools.pm

```diff
@@ -0,0 +1,42 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Simple vmware client testing with updated open-vm-tools
+# Maintainer: Michael Grifalconi <mgrifalconi@suse.com>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use version_utils;
+use xen;
+
+sub run {
+    my ($self) = @_;
+
+    assert_script_run "ssh root\@$_ zypper -n in open-vm-tools" foreach (keys %xen::guests);
+
+    validate_script_output "ssh root\@$_ /usr/bin/vmware-checkvm", sub { m/good/ } foreach (keys %xen::guests);
```

Comment:
I prefer `assert_script_run ' … | grep …';` but this way is I'd say more native. What is your opinion, @grisu48 ?

Reply:
I also think so. But unless it causes problems I think this solution is ok

Reply:
Hmmm, `validate_script_output` is used a lot here. I agree with @pdostal, that `assert_script_run ' … | grep …';` might be the more sane way for virtualization runs as it turned out to be more robust.

I would say Michael can decide what he thinks works better here. `validate_script_output` might look a bit nicer, the other approach might be a bit more stable :slightly_smiling_face:

Reply:
If you say it's more stable that way, I agree to change it. I simply used the already available function.
But then I would like to adapt the official documentation and mention that is discouraged to use for virtualization topics http://open.qa/api/testapi/#_validate_script_output
Otherwise no one will know about this problem.

Reply:
This is probably for bigger discussion we might be wrong as well.